### ISSUE TITLE
Niedrigere Preise

### ISSUE THREAD

**Issue by Justman10000** [dwwe2017/teamspeak-interface#6]
I would suggest lowering the prices! Sorry, but for a manage web dashboard for TeamSpeak, which felt has a design from the 1990s, the prices are hard (although this would actually be the case in any case)...

For example, prices could be as follows:

STARTER License = 1$
GAMING License = 2,5 $
HOSTING License = 5 $

I don't will or want saying it is easy or takes little to no time to write such an application, but one should be aware that lower prices bring more purchases (according to the motto: Great software, and so cheap, I'll take 5 licences right away...) One understands 🙂

**Comment by Justman10000** [comment]
Or even better, one sets this for lifetime:

STARTER License = 6,99 $
GAMING License = 14,99 $
HOSTING License = 19,99 $

**Comment by dwwe2017** [comment]
Hello and thank you for your contribution. I'll definitely think about it sometime. Which paid competitor products with such a low price can you tell me for comparison?

**Comment by Justman10000** [comment]
> Hello and thank you for your contribution. I'll definitely think about it sometime. Which paid competitor products with such a low price can you tell me for comparison?

Hmm, [TS3 Manager](https://www.ts3.app) comes to mind! It's even open source, although it doesn't have a hosting version with a reselling system, but one could perhaps take this as inspiration, also with regard to the design of your TSI (as I said, it feels like it's from the 90s 😅)

**Comment by dwwe2017** [comment]
Which design do you mean? I don't think the Hurricane template is that '90s at all? Of course, I agree with you about the standard :) Nothing against the TS3 Manager, it's definitely a great program, but in terms of function, there's a world of difference in my opinion. Nevertheless, I will consider the prices.

**Comment by dwwe2017** [comment]
> Or even better, one sets this for lifetime:
> 
> STARTER License = 6,99 $ GAMING License = 14,99 $ HOSTING License = 19,99 $

There definitely won't be

**Comment by dwwe2017** [comment]
> I would suggest lowering the prices! Sorry, but for a manage web dashboard for TeamSpeak, which felt has a design from the 1990s, the prices are hard (although this would actually be the case in any case)...
> 
> For example, prices could be as follows:
> 
> STARTER License = 1$ GAMING License = 2,5 $ HOSTING License = 5 $
> 
> I don't will or want saying it is easy or takes little to no time to write such an application, but one should be aware that lower prices bring more purchases (according to the motto: Great software, and so cheap, I'll take 5 licences right away...) One understands 🙂

Possibly double, but I'll find out.

**Comment by Justman10000** [comment]
> > Or even better, one sets this for lifetime:
> > STARTER License = 6,99 $ GAMING License = 14,99 $ HOSTING License = 19,99 $
> 
> There definitely won't be

Why not? We all know that most people hate subscriptions, and rightly so

**Comment by Justman10000** [comment]
> > I would suggest lowering the prices! Sorry, but for a manage web dashboard for TeamSpeak, which felt has a design from the 1990s, the prices are hard (although this would actually be the case in any case)...
> > For example, prices could be as follows:
> > STARTER License = 1$ GAMING License = 2,5 $ HOSTING License = 5 $
> > I don't will or want saying it is easy or takes little to no time to write such an application, but one should be aware that lower prices bring more purchases (according to the motto: Great software, and so cheap, I'll take 5 licences right away...) One understands 🙂
> 
> Possibly double, but I'll find out.

I thought about that too, but it would be too much, 10$ a month for one application? I mean, you can afford Netflix for that 🤣

**Comment by Justman10000** [comment]
@dwwe2017

**Comment by dwwe2017** [comment]
> > > I would suggest lowering the prices! Sorry, but for a manage web dashboard for TeamSpeak, which felt has a design from the 1990s, the prices are hard (although this would actually be the case in any case)...
> > > For example, prices could be as follows:
> > > STARTER License = 1$ GAMING License = 2,5 $ HOSTING License = 5 $
> > > I don't will or want saying it is easy or takes little to no time to write such an application, but one should be aware that lower prices bring more purchases (according to the motto: Great software, and so cheap, I'll take 5 licences right away...) One understands 🙂
> > 
> > 
> > Possibly double, but I'll find out.
> 
> I thought about that too, but it would be too much, 10$ a month for one application? I mean, you can afford Netflix for that 🤣

Yes, and if you are a hoster who offers the panel to its customers, you have priceless added value.

**Comment by Justman10000** [comment]
> > > > I would suggest lowering the prices! Sorry, but for a manage web dashboard for TeamSpeak, which felt has a design from the 1990s, the prices are hard (although this would actually be the case in any case)...
> > > > For example, prices could be as follows:
> > > > STARTER License = 1$ GAMING License = 2,5 $ HOSTING License = 5 $
> > > > I don't will or want saying it is easy or takes little to no time to write such an application, but one should be aware that lower prices bring more purchases (according to the motto: Great software, and so cheap, I'll take 5 licences right away...) One understands 🙂
> > > 
> > > 
> > > Possibly double, but I'll find out.
> > 
> > 
> > I thought about that too, but it would be too much, 10$ a month for one application? I mean, you can afford Netflix for that 🤣
> 
> Yes, and if you are a hoster who offers the panel to its customers, you have priceless added value.

?
